src/internal/operations/restore.go

```diff
@@ -167,6 +168,11 @@ func (op *RestoreOperation) Run(ctx context.Context) (restoreDetails *details.De
 	if err != nil {
 		// No return here!  We continue down to persistResults, even in case of failure.
 		logger.CtxErr(ctx, err).Error("running restore")
+
+		if strings.Contains(err.Error(), "no restore path given") {
```

Comment:
checking error strings is a smell since the check is brittle and prone to failure when the error message changes.  Is there a way to check for this using errors.Is instead?

Reply:
as of my understanding errors.Is checks for equality. And here the check is required for sub-string. Is there a way to just check the sub-string and not complete string? Also does clues package has anything to do this check?

Reply:
Looking into it. Somehow errors.is function isn't able to match the case.

Reply:
Is there a way for us to wrap the original error with something in corso so that it could match using errors.Is.

Reply:
clues package itself has a `errors.Is` func which should ideally take care of the stacks too. But is unable to compare the error even when the messages are same. 
Not sure about the reason. Maybe metadata is different.

Reply:
Done